desc/objectives/linear_objectives.py

```diff
@@ -3238,3 +3234,393 @@ def compute(self, params, constants=None):
 
         """
         return params["Psi"]
+
+
+class FixNearAxisR(_FixedObjective):
+    """Fixes an equilibrium's near-axis behavior in R to specified order.
+
+        NOTE: does not also fix the axis R , for that
+        use the FixAxisR objectives.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium that will be optimized to satisfy the Objective.
+    nae_eq : Qsc, optonal
```

Comment:
would it make more sense to combine this with `target`?

Reply:
hm, yea maybe, so that target is either None or qsc_eq